modules/codemod/lib/v6/utils/index.ts

```diff
@@ -179,7 +159,8 @@ export function renameImports(
   api: API,
   root: Collection<any>,
   packageName: string,
-  specifierMap: Record<string, string>
+  specifierMap: Record<string, string>,
+  discoveredImportAliases: Record<string, boolean> = {}
```

Comment:
Adding this not only fixes the `Header` import problem, but means we address a lot of possible overlap with non-Canvas Kit components that share the same name.